state/runtime/precompiled/precompiled.go

```diff
@@ -36,6 +39,22 @@ func init() {
 	abiBoolFalse = encodedBool
 }
 
+func init() {
+	ActivePrecompiles = []types.Address{
+		types.StringToAddress("1"),
+		types.StringToAddress("2"),
+		types.StringToAddress("3"),
+		types.StringToAddress("4"),
+		types.StringToAddress("5"),
+		types.StringToAddress("6"),
+		types.StringToAddress("7"),
+		types.StringToAddress("8"),
+		types.StringToAddress("9"),
+		types.StringToAddress(contracts.NativeTransferPrecompile.String()),
+		types.StringToAddress(contracts.BLSAggSigsVerificationPrecompile.String()),
+	}
+}
```

Comment:
To ensure a clean code structure, I separated these two into two different init functions.
Will merge these init into one.

Also do you think it would be better to take precompiles address from `Precompiled struct` for each transactions rather than initializing a new variable i.e `ActivePrecompiles`

Reply:
> Also do you think it would be better to take precompiles address from Precompiled struct for each transactions rather than initializing a new variable i.e ActivePrecompiles

Yep, I think it would be better. You could add a new slice called `contractAddrs []types.Address` to `Precompiled` struct and populate it lazily (every time the `register` function is being called). It is better that way than to have to worry about maintaining two data structures that have redundant data.